packages/flame/lib/src/collisions/hitboxes/composite_hitbox.dart

```diff
@@ -15,6 +15,7 @@ class CompositeHitbox extends PositionComponent
     super.size,
     super.scale,
     super.angle,
+    super.nativeAngle,
```

Comment:
@spydon Do we want to add these new constructor parameters everywhere? They are accessible through a setter anyway.

Reply:
Hmm, not sure, let me check with the rest.

Reply:
I would like it to be available in at least these components:
- `SpriteComponent`
- `SpriteGroupComponent`
- `SpriteAnimationComponent`
- `SpriteAnimationGroupComponent`

*Just a personal preference 😄

Reply:
Maybe that's a good compromise, let's go with that.

Reply:
Done. Reduced the changes to sprite related components.